src/geo/transform.js

```diff
@@ -315,7 +315,7 @@ class Transform {
 
         // Compute zoom level from the height of the camera relative to the terrain
         const cameraZoom: number = this._cameraZoom;
-        const elevationAtCenter = this._elevation.getAtPoint(MercatorCoordinate.fromLngLat(this.center));
+        const elevationAtCenter = this._elevation.getAtPoint(MercatorCoordinate.fromLngLat(this.center), 0) || 0;
```

Comment:
I'm not sure that the `getAtPoint(..., 0) || 0` are strictly necessary. We're calling it with `defaultIfNotLoaded` set to 0, so `null` is not an option, am I missing something?

Reply:
yea its dead code, its used to keep flow happy, because flow has no way of knowing that the default value gurantees the type.